R/fb_league_stats.R

```diff
@@ -36,34 +36,33 @@
       .frequency = "once",
       .frequency_id = "fb_league_stats-player"
     )
+
     session <- worldfootballr_chromote_session(url)
-    page <- worldfootballr_html_page(session)
+    player_table <- worldfootballr_html_player_table(session)
```

Comment:
the new solution is a little more specific. this function only returns the HTML for 1 table, while the prior solution returned the HTML for all (3) tables on the page, from which the player table was later plucked. arguably this new solution is better